crates/dbsp/src/storage/backend/mod.rs

```diff
@@ -323,3 +278,67 @@ impl StorageCacheFlags for OpenOptions {
         self
     }
 }
+
+/// Maximum number of buffers that system calls accept in one operation.
+///
+/// We only use multibuffer system calls on Linux, so the value is arbitrary
```

Comment:
maybe should be 0 so things break(?) in case it ever gets used on non-linux in the future